appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/model/BaseAppyxComponent.kt

```diff
@@ -254,10 +254,9 @@ open class BaseAppyxComponent<InteractionTarget : Any, ModelState : Any>(
     }
 
     override fun onDragEnd() {
-        if (!_isAnimating.value) {
-            drag.onDragEnd()
-            settle(gestureSettleConfig)
-        }
+        // TODO sometimes drag is started in animated mode and can not be ended
```

Comment:
Can you please elaborate when this could happen?

Reply:
Dragging animates GenericFloat in SpotlightSlider, there's a condition that if it's being animated don't call onDragEnd, but it's wrong. I think it should be revised

Reply:
Cool... My approach also fix that.